docs/BuildFromSource.md

```diff
@@ -224,6 +224,8 @@ code .
 
 When developing in VS Code, you'll need to use the `build.cmd` or `build.sh` scripts in order to build the project. You can learn more about the command line options available, check out [the section below](using-dotnet-on-command-line-in-this-repo).
 
+After navigating to the parent directory of the desired project, you can run the build script for that individual project.
```

Comment:
Does "parent directory" here mean something like "closest ancestor directory that contains a `build.cmd`/`build.sh` file"? Many of the `.csproj` files don't have a build script in their immediate parent directory.

Reply:
Correct -- I've added some more text/samples here to explain the relationship between the build scripts and the project modified.